graphql_tests/postWinnersTest.js

```diff
@@ -0,0 +1,9 @@
+const postWinners = require("../graphql/postWinners.js");
+const should = require("chai").should();
+
+describe("postWinners", function() {
+  it("should return true for adding user to db", async function() {
+    response = await postWinners(["nash"]);
```

Comment:
a hacky way to make sure this doesn't affect prod database since we don't have a test one is to add a test method that will remove a win from the user with handle `nash`